elpy.el

```diff
@@ -2261,7 +2261,8 @@ Also sort the imports in the import statement blocks."
           (elpy-importmagic-add-import (car choice))))))
   ;; now get a new import statement block (this also sorts)
   (let* ((res (elpy-rpc "remove_unreferenced_imports" (list buffer-file-name
-                                                            (elpy-rpc--buffer-contents)))))
+                                                            (elpy-rpc--buffer-contents)
+                                                            elpy-project-root))))
```

Comment:
Shouldn't this use (elpy-project-root) to calculate the root if there is none yet?